src/workerd/server/server.c++

```diff
@@ -3087,18 +3088,47 @@ kj::Own<Server::Service> Server::makeWorker(kj::StringPtr name,
   auto observer = kj::atomicRefcounted<IsolateObserver>();
   auto limitEnforcer = kj::refcounted<NullIsolateLimitEnforcer>();
 
+  // Create the FsMap that will be used to map known file system
+  // roots to configurable locations.
+  // TODO(node-fs): This is set up to allow users to configure the "mount"
```

Comment:
Nit: I would remove that from a TODO() and wait for someone to actually request the feature.
TODOs have a tendency to mount up and shouldn't document possible feature requests in my opinion but rather things that we think we should do eng-wise.

Reply:
Kenton has specifically indicated that he'd like to see this, so I think that qualifies here?

Reply:
For the time being, I'll leave this as a TODO